.gitlab-ci.yml

```diff
@@ -241,3 +241,39 @@ integration_mariadb:
     - name: mariadb
       alias: sqlserver
 
+php-cs-fixer:
+  image: 
+    name: cytopia/php-cs-fixer
+    entrypoint: [""]
+  artifacts:
+    when: always
+    reports:
+      codequality: code-quality.json
+  script:
+    - /usr/bin/php-cs-fixer fix --format=gitlab --dry-run --diff . > code-quality.json || true
+
+php linter:
+  stage: test
+  image: pipelinecomponents/php-linter:latest
+  script:
+    - parallel-lint --colors .
+
+phpcs PSR12:
+  stage: test
+  image: pipelinecomponents/php-codesniffer:latest
+  script:
+    - phpcs -s -p --colors --extensions=php --standard=PSR12 .
+
+php-compatibility 7.3:
```

Comment:
Shouldn't this be 7.2? I'm also fine just calling it `php-compatibility`

Reply:
The copied stuff had 7.3, will indeed change it.